example/life_game/runweb.ml

```diff
@@ -0,0 +1,30 @@
+open Lwt
```

Comment:
we should probably use [tiny_httpd](https://github.com/c-cube/tiny_httpd/blob/master/tiny_httpd.opam) instead of cohttp